.travis.yml

```diff
@@ -22,14 +23,14 @@ matrix:
     fast_finish: true
     include:
           # Minimum supported Symfony version and lowest PHP version
-        - php: 7.2
-          env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" SYMFONY_DEPRECATIONS_HELPER="weak"
+        - php: 7.3
+          env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" SYMFONY_DEPRECATIONS_HELPER="weak" DEPENDENCIES="php-http/guzzle7-adapter:^0.1.1"
```

Comment:
i suggest to use guzzle6-adapter for the lowest version build. i think we only added guzzle7 to discovery later - but we should not raise the minimum discovery component requirement, it does work together (just not with guzzle7)

Reply:
Ah, thanks for the hint!